appendices/scope.md

```diff
@@ -0,0 +1,72 @@
+## Python package domain scope
+
+The following categories are the current domain areas that fall into the 
+pyOpenSci domain scope.
+
+- **Data retrieval:** Packages for accessing and downloading data from online sources. Includes wrappers for accessing APIs.
+- **Data extraction:** Packages that aid in retrieving data from unstructured sources such as text, images and PDFs.
+- **Data munging:** Tools for processing data from scientific data formats.
+- **Data deposition:** Tools for depositing data in scientific research repositories.
+- **Reproducibility:** Tools to scientists ensure that their research is reproducible. E.g. version control, automated testing, or citation tools.
+- **Geospatial:** Packages focused on the retrieval, manipulation, and analysis of spatial data.
```

Comment:
@lwasser Why is Geospatial a particular category? The others are very `generic`, except for this one.

Reply:
good question @arianesasso we added that because it made a lot of sense for the organization to start out with a focus in areas that are my expertise given i'm setting up structures. This doesn't mean that we dont review other things but that is why this is there.  Now this was 3+ years ago (i can check but i think i started peer review in 2018) SO we could re-evaluate it it could be time? so glad you noted this it's been like this for a long time :) fresh eyes are good.